internal/handlers/pg/pgdb/pool.go

```diff
@@ -199,13 +199,19 @@ func (pgPool *Pool) SchemaStats(ctx context.Context, schema, collection string)
                                          AND s.relname = t.table_name
       LEFT OUTER
       JOIN pg_indexes                AS i ON i.schemaname = t.table_schema
-                                         AND i.tablename = t.table_name
-     WHERE t.table_schema = $1`
+                                         AND i.tablename = t.table_name`
 
-	args := []any{schema}
-	if collection != "" {
-		sql = sql + " AND t.table_name = $2"
-		args = append(args, collection)
+	// TODO should we exclude service schemas and tables?
```

Comment:
@AlekSi, if this func is called for `serverStatus` (so we calculate the stats across the whole DB), should we exclude any service schemas/tables? Smth like ` WHERE  t.table_schema != 'magic_ferret_service_schema'?

Reply:
What is service schema?

For tables – yes, we should skip tables that start with `reservedPrefix`.

Not sure if we want to do in that PR

Reply:
@AlekSi well, it seems to me that `reservedPrefix` is used for both cases with db and collection creation. With that, it seems that there are some "service schemas" or something :D

Reply:
Reworded everything and fixed TODO.